test/integration/patients/worklist/worklist.js

```diff
@@ -2555,15 +2555,15 @@ context('worklist page', function() {
           type: 'patients',
           attributes: {
             first_name: 'Patient',
-            last_name: 'Field 1',
+            last_name: 'Field A',
           },
           relationships: { 'patient-fields': { data: [{ id: '1' }] } },
         });
 
         fx.included.push({
           id: '1',
           type: 'patient-fields',
-          attributes: { value: { value: 1 }, name: 'foo' },
```

Comment:
Reverted these tests back to before: https://github.com/RoundingWell/care-ops-frontend/pull/879.

We'll need to update the tests to get coverage back to 100%. Will do that in separate card.

Reply:
Sorry missed the comment. I'll approve.